docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc

```diff
@@ -0,0 +1,66 @@
+[id='glusterfs-reconfig-proc']
+= Changing GlusterFS configuration
+
+You must check whether your OpenShift environment uses GlusterFS to provide permanent storage volumes. If it uses GlusterFS, you must change the GlusterFS configuration.
```

Comment:
you must change the GlusterFS configuration -> you should tune your GlusterFS storage